app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="A scientific editing platform designed to create beautiful technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for editing \
+structured documents with different types of content: text, mathematics, graphics, \
+interactive content, slides, etc.\
+Its rendering engine uses high-quality typesetting algorithms so as to produce \
+professionally looking documents, which can either be printed out or presented from \
+a laptop."
```

Comment:
Didn't focus on this yet :)
Maybe for the SUMMARY: "Wysiwyg text processor with high-quality maths" (from gentoo http://gpo.zugaina.org/app-office/texmacs) (we like summary to be short) :)
For DESCRIPTION, some lines cross the 80 char limit see: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips

Reply:
I'm not very keen of that description. I shortened ours.

Reply:
What's wrong with using yours, if so you can change that here, keeping in mind the 80 char limit

Reply:
Maybe use the description from https://github.com/texmacs/texmacs ?

Reply:
I've rephrased taking into account the description in wikipedia. The important thing is the "structured" feature, and technically is not more WYSIWYG then WYSIWYM. I think I should change that readme :)